src/components/TextCard/TextCard.tsx

```diff
@@ -40,8 +41,10 @@ export const TextCard = ({
   id,
   metaLabel,
   title,
+  titleAs = 'h3',
   type
 }: TextCardProps) => {
+  const Element = titleAs;
```

Comment:
Call this `TitleElement` to be consistent with other components that use this prop

Reply:
updated :)